Utilities/FileIO.hpp

```diff
@@ -251,13 +252,24 @@ void readMatPowerBaseMVA(SystemModelData<RealT, IdxT>& mp, std::string& line)
 
 namespace GridKit
 {
+
+/**
+ * @brief Reads a file in for tabulated data
+ *
```

Comment:
It's not too clear what this functions expects as the files data format. Perhaps clarify here on this.

Reply:
Updated comments to clarify that.